src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -3622,19 +3622,10 @@ class SymbolTableVisitor : public CommonVisitor<SymbolTableVisitor> {
             in any directory other than src/runtime will also
             be ignored.
         */
-        if (mod_sym == "lpython") {
+        if (mod_sym == "lpython" || mod_sym == "lptypes" || mod_sym == "numpy") {
```

Comment:
I uploaded the `lpython.cpp` to `pypi` with the name `lptypes` (`lpython` is not available, also `ltypes` is not available). I would like to know if the name `lptypes` is fine or if we should use any other name.

Reply:
I see, we need a CPython package, for compatibility. I would call the Python module lpython. For now I would distribute it with the "lpython" compiler, say the conda package. For pypi, maybe "lpython_emulation" pypi package would work? It would still install lpython.py.

Reply:
Got it. 

One concern:
>For now I would distribute it with the "lpython" compiler, say the conda package

Every package that we develop uses the `i32/i64/...` types from `lpython`. Do you mean to package the `lpython.py` and `goto.py` (since `lpython.py` imports `with_goto` from `goto.py`) with every package that we upload to `pypi` (and `conda`)?

Since every package almost always uses types from `lpython`, I think it would not work if its dependencies are not included or the dependencies do not already exist on `pypi`.